lib/ansible/modules/cloud/amazon/ec2_vpc_net.py

```diff
@@ -149,7 +149,7 @@ def update_vpc_tags(vpc, module, vpc_obj, tags, name):
     tags.update({'Name': name})
     try:
         current_tags = dict((t.name, t.value) for t in vpc.get_all_tags(filters={'resource-id': vpc_obj.id}))
-        if cmp(tags, current_tags):
+        if lambda tags, current_tags: (tags > current_tags) - (tags < current_tags):
```

Comment:
Just change this to `==` since it's a single-layer dictionary of hashable types.